src/parser.cc

```diff
@@ -1131,7 +1131,8 @@ bool readXml(TiXmlElement *_xml, ElementPtr _sdf, Errors &_errors)
           else
           {
             // Add default element
-            _sdf->AddElement(elemDesc->GetName());
+            ElementPtr default_element = _sdf->AddElement(elemDesc->GetName());
```

Comment:
nit: We should use `camelCase` per the style [guide](https://ignitionrobotics.org/docs/all/contributing#style-guides)

Reply:
Done in 52cd85f7c78a366d72b266a842c372c48f1588ce